.github/workflows/goreleaser.yaml

```diff
@@ -60,7 +59,8 @@ jobs:
           PKG: github.com/operator-framework/operator-lifecycle-manager
 
       - name: Generate quickstart release manifests
-        run: make release ver=${{ env.IMAGE_TAG }} IMAGE_REPO=quay.io/operator-framework/olm
+        if: startsWith(github.ref, 'refs/tags')
```

Comment:
Adding this condition back (was removed in #3293). This step pulls an image and we don't have an image when running against anything other than a tag.